package/firmware/ipq-wifi/Makefile

```diff
@@ -28,6 +28,7 @@ endef
 # <https://wireless.wiki.kernel.org/en/users/drivers/ath10k/boardfiles>
 
 ALLWIFIBOARDS:= \
+	arcadyan_aw1000 \
```

Comment:
Did you make a PR for this BDF?

Cause this wont work as its not present in the BDF repo.

Reply:
I will, I'm just wondering about qmi-board-id changed itself.

Reply:
Well, it cannot really change itself as the FW will try and read the fuse on boot.